test/Npgsql.Tests/Types/RangeTests.cs

```diff
@@ -12,7 +12,33 @@ namespace Npgsql.Tests.Types
     /// </remarks>
     class RangeTests : TestBase
     {
-        [Test, NUnit.Framework.Description("Resolves a range type handler via the different pathways")]
+        #region Issues
+
+        [Test]
+        [IssueLink("https://github.com/npgsql/npgsql/issues/1943")]
+        public void Roundtrip_canonicalization_causes_issues()
```

Comment:
I'm not sure of the value of having this test... The issue seems to be enough for tracking this no?

Reply:
I had [this test](https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/blob/1bfb584141019d97391a7d6902400b42b04c6b24/test/EFCore.PG.FunctionalTests/Query/ArrayQueryTest.cs#L139-L149) in mind when I added it:

```c#
[Fact(Skip="https://github.com/aspnet/EntityFramework/issues/9242")]
public void Length_on_EF_Property()
{
    using (var ctx = CreateContext())
    {
        // TODO: This fails
        var x = ctx.SomeEntities.Single(e => EF.Property<int[]>(e, nameof(SomeArrayEntity.SomeArray)).Length == 2);
        Assert.Equal(new[] { 3, 4 }, x.SomeArray);
        AssertContainsInSql(@"WHERE array_length(e.""SomeArray"", 1) = 2");
    }
}
```

I don't have a strong opinion either way. I can drop this in the next commit, just let me know.

Reply:
As you wish, I don't have a strong opinion either way either :)